lib/src/sv_common.c

```diff
@@ -158,6 +158,20 @@ sign_or_verify_data_free(sign_or_verify_data_t *self)
   free(self);
 }
 
+// Convert Unix timestamp (seconds since 1970) to 1601-based timestamp
+void
+convert_unix_to_1601(int64_t *timestamp)
+{
+  *timestamp = (*timestamp + EPOCH_DIFF) * HUNDRED_NANOSECONDS;
```

Comment:
The Axis Unix Epoch is in microseconds, not seconds. I see that the information is missing in signed_video_sign.h, but present in signed_video_auth.h